Makefile

```diff
@@ -10,16 +10,16 @@ test:
 bench:
 	go test -v ./... -bench=. -timeout 2m
 
-run:
-	cd vflow; go run $(GOFILES) -sflow-workers 100 -ipfix-workers 100
```

Comment:
This is somethign that gets weird by having the logic of go within the Makefile, and not amking use of the go build tools: 

```
cd vflow; go run vflow.go ipfix.go ipfix_windows.go ipfix_unix.go sflow.go netflow_v9.go options.go stats.go  -sflow-workers 100 -ipfix-workers 100
# command-line-arguments
./ipfix_unix.go:31: mirrorIPFIXDispatcher redeclared in this block
	previous declaration at ./ipfix_windows.go:25
make: *** [run] Error 2
```
This happens becuase the build controls don't over write your manually adding the file to the cli.  So the only way I can think of making this work is pushing the logic into the Makefile (gross) or using `go build` and then running with CLI? 

Happy to do what ever you think is correct.